Image/source/image.css

```diff
@@ -0,0 +1,17 @@
+.image-container {
```

Comment:
Consider keeping these styles in the different stylesheet as these are more related to the documentation site if I am not wrong.